releasenotes/notes/add-protocol-classification-to-npm-2a50967a3d70ced9.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Network Performance Monitoring now support protocol classification, which labels each connection with a L7 protocol.
```

Comment:
Keep in mind that reading this would imply that this already works end to end, which it doesn't since the backend support hasn't been added.

Reply:
Good catch
Is something like "adding required infrastructure to enable protocol classification in the future" better!